RELEASE.md

```diff
@@ -98,17 +98,21 @@ $ git push
 # Once PR against openshift/release repo is merged, the CI is setup for release-branch
 ```
 
-### Update Dockerfile in release branch with correct version:
+### Update Dockerfiles and RPM SPEC file in release branch with correct version:
 ```bash
-# Update Dockerfile with current release number:
+# Update Dockerfiles namely: Dockerfile.rhel and Dockerfile.cliartifacts.rhel with current release number:
 # - Change value of tag to current release (RUN TAG="v0.14.0" make build)
 # - Change value of LABEL 'version' to current release (v0.14.0)
-$ vi Dockerfile
+$ vi Dockerfile.rhel
+$ vi Dockerfile.cliartifacts.rhel
```

Comment:
@warrenvw : FYI - We'd need renaming these Dockerfiles after sync at their respective dist git repos too.